transformations/suspecting_paraphraser/transformation.py

```diff
@@ -0,0 +1,192 @@
+from typing import Tuple, List
+import nltk
+from pyinflect import getInflection
+import spacy
+import gender_guesser.detector as gender
+from enum import Enum
+import numpy as np
+
+
+from interfaces.QuestionAnswerOperation import QuestionAnswerOperation
+from tasks.TaskTypes import TaskType
+
+"""
+Base Class for implementing the different input transformations a generation should be robust against.
+"""
+
+
+class Pronouns:
+    HE = 0
+    SHE = 1
+    IT = 2
+    THEY = 3
+
+
+class SuspectingParaphraser(QuestionAnswerOperation):
```

Comment:
Can you please write 2-3 lines of comment for this class and also in a function wherever requires? So that any user will get proper clarity instead of going through the whole code.